client/clip_client/client.py

```diff
@@ -436,3 +448,47 @@ async def arank(self, docs: Iterable['Document'], **kwargs) -> 'DocumentArray':
                 )
 
         return results
+
+    def index(self, content: Iterable['Document'], **kwargs):
+        """Index the embeddings created by server CLIP model.
+        Given the document with embeddings, this function create an indexer which index
```

Comment:
The docstr is not corrct. `index` and `query` functions both accept raw text/image docs without embeddings.

Reply:
What's more, we also implment async version, `aindex`, `aquery`

Reply:
The `index` should have the same function signature as `encode`